pkg/ddevapp/ddevapp.go

```diff
@@ -2838,6 +2838,7 @@ func GetActiveAppRoot(siteName string) (string, error) {
 // To use the current working directory, siteName should be ""
 func GetActiveApp(siteName string) (*DdevApp, error) {
 	app := &DdevApp{}
+	siteName = NormalizeProjectName(siteName)
```

Comment:
If we do this only for the enclosing directory, this function should be called only if `siteName == ""`, right?

Reply:
`NormalizeProjectName` should probably be called somewhere here?:

https://github.com/ddev/ddev/blob/28a5afcdd56cc190aca33303fc1405c0c8ebabe2/pkg/ddevapp/ddevapp.go#L2791-L2804

> When getting the project name from the directory name, underscores are replaced with hyphens.

Actually replacing underscores for the directory name only makes it more complex, than we replace underscores in all cases.

Reply:
AFAICT GetActiveAppRoot() (if called with siteDir) is using the registration info from project_list.yaml, so doesn't need to do anything with NormalizeProjectName()

Reply:
So we don't even need to call it here?

Reply:
I guess that's probably true.